spec/models/invoice_spec.rb

```diff
@@ -5,6 +5,23 @@
     it {should define_enum_for(:status).with ['in progress', 'completed', 'cancelled'] }
```

Comment:
we ended up changing `with` to `with_values` in ours to get rid of that deprecation warning

Reply:
Noted, will change ours for consistency